core/shortcut_registry.ts

```diff
@@ -219,24 +219,36 @@ export class ShortcutRegistry {
   /**
    * Handles key down events.
    *
+   * - Any `KeyboardShortcut`(s) mapped to the keyscode that cause
+   *   event `e` to be fired will be processed, in order from least-
+   *   to most-recently registered.
+   * - If the shortcut's `preconditionFn` exists it will be called,
```

Comment:
Slight rephrasing to reduce ambiguity about "it":
- If the shortcut's `preconditionFn` exists it will be called. If `preconditionFn` returns false the shortcut's `callback` function will be skipped. Processing will continue with the next shortcut, if any.